rosdep/base.yaml

```diff
@@ -3040,6 +3044,7 @@ libglew-dev:
   gentoo: [media-libs/glew]
   nixos: [glew]
   openembedded: [glew@openembedded-core]
+  opensuse: [glew-devel]
```

Comment:
The main link didn't find the devel version but it appears to be in the spec file. https://build.opensuse.org/package/view_file/openSUSE:Leap:15.2/glew/glew.spec?expand=1

Reply:
I guess I accidentally posted just the package view link. I've been posting the binary repository links attached to the package which for glew would have been ...

https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/glew/standard